src/comment.test.js

```diff
@@ -0,0 +1,16 @@
+/** * @jest-environment jsdom */
+import { countComments } from './involvementAPI';
+
+global.fetch = jest.fn(() => Promise.resolve(
+  {
+    json: () => Promise.resolve(new Array(3)),
+  },
+));
+
+describe('Testing comment count', () => {
+  document.body.innerHTML = '<div></div>';
+  it('Comments count should be 7', async () => {
+    const count = await countComments();
+    expect(count).toBe(3);
+  });
+});
```

Comment:
Kindly look at this requirement:

![image](https://user-images.githubusercontent.com/57421823/152525745-d2a6331d-3c06-428b-96d4-4b05d1364ffd.png)

Therefore, 

- [ ] kindly please make functions to calculate your counters (comments and movies), and please don't use the API to give you those numbers.

- [ ] You will need to update your tests, so they check those functions' correct functionality.

That way we will meet that requirement and get closer to approval 💪